src/app/core/components/breadcrumbs/breadcrumbs.component.spec.ts

```diff
@@ -1,4 +1,7 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+
+import { BreadcrumbsService } from '@app/services/breadcrumbs/breadcrumbs.service';
```

Comment:
consider simplifying imports.

Reply:
@boale, what do you mean by simplifying imports in this case?

Reply:
something like `import { BreadcrumbsService } from '@app/services';`

Reply:
Okay